### ISSUE TITLE
npm package v3.1.0 does not includes last changes

### ISSUE THREAD

**Issue by simonbruneaud** [meltingice/psd.js#91]
The last version of the psd.js module does not include some features like artboards.
Is it possible to update the npm package?

Best regards.

**Comment by fabulousduck** [comment]
Could you elaborate a bit more ?
After doing some testing with artboards, i can confirm that they do still work and parse as they should?

**Comment by joliejuke-orga** [comment]
I have this structure for instance:

```
--- Artboard 1
------ group button
------------ Text
------------ Shape background
```

I parse each node using a recursive function and I always call for the method:
`node.get('artboard')` but I always got 'undefined'.
Maybe I'm not using this function correctly?

**Comment by fabulousduck** [comment]
From what i can see, an artboard is a layer type by itself ( @meltingice will need to confirm this as i am not 100% sure as my coffeescript is shoddy at best ). so using `.get()` on it wont work, since that is for getting values or helpers on a layer itself. 

Maybe this helps you out ?

**Comment by fabulousduck** [comment]
After some more testing, i get inconsistent results getting artboard data using `node.get()`. i will mark this as an issue now as this is something that will need to be fixed soon. For now i can't come up with a solution for you, maybe @meltingice has any thoughts on this ?

**Comment by joliejuke-orga** [comment]
Thanks for confirmation.
The weird thing is that Artboards don't work on npm but it work with the github repository.
Maybe the npm package is not up to date?

**Comment by fabulousduck** [comment]
This is indeed true after comparing the 2. The artboard file is not present in the NPM package version.
@meltingice i do think this is rather urgent as artboards are an important feature, so could you take a look at updating the npm repo when you get the chance ?

**Comment by meltingice** [comment]
Yeah, it looks like npm hasn't been updated since some PRs were merged. Updating the npm version now.

**Comment by meltingice** [comment]
Okay `psd@3.2.0` is released. Let me know if that fixes your issues.

**Comment by fabulousduck** [comment]
After installing version `~3.2.0` i can confirm that the `artboard.coffee` is now present.
